### PR TITLE
[Sitemap] Add color 'itemValue'

### PR DESCRIPTION
**Pull request by mueller-ma**
https://github.com/openhab/openhab-core/pull/3453

### CODE REVIEW

**Comment by mueller-ma** [comment]
Please note that the feature hasn't been implemented yet, but the PR is in review.

File: ui/sitemaps.md [hunk 1]
```diff
@@ -677,8 +677,13 @@ Generally, you can expected that valid HTML colors will be accepted (e.g. `green
...
 
-For better theming support Basic UI and the Android app also accepts the keywords `primary` and `secondary` as colors.
-They will be mapped to the corresponding colors of the UI theme.
+There are also the following keywords that can be used as colors:
+
+| Color Name | Note                      |
+|------------|---------------------------|
+| primary    | Primary theme color       |
+| secondary  | Secondary theme color     |
+| itemValue  | The state of a color item |
```
Comment (lolodomo) [inline ui/sitemaps.md:686]: You could also mention that this color will be ignored if the sitemap element has no item defined or if the item state is not a color.

**Comment by mueller-ma** [comment]
The feature has been merged.

**Review by stefan-hoehn — APPROVED** [review]
Dependent PR has been merged by now. Doc changes look good to me.

**Pull request merged**